src/redis.ts

```diff
@@ -106,9 +106,8 @@ class buildRedisStoreWithConfig implements RedisStore {
     return await this.redisCache.ttl(key);
   };
 
-  public async scan(pattern: string): Promise<ScanReply> {
-    let cursor = 0;
-    return await this.redisCache.scan(cursor, {MATCH: pattern});
+  public async scan(pattern: string, cursor: number = 0, count?: number): Promise<ScanReply> {
```

Comment:
```suggestion
  public async scan(pattern: string, cursor: number = 0, count = 10): Promise<ScanReply> {
```
let's use 10 as the default 
<img width="695" alt="image" src="https://github.com/blindspot-security/node-cache-manager-redis-store/assets/57172661/756052c3-e513-4719-b2b4-98161a5f256d">

Reply:
https://github.com/blindspot-security/node-cache-manager-redis-store/pull/11/commits/b8a12905ae5ee790cadc902d82b9213a6701a06e

Reply:
thanks